### PR TITLE
Doc'd archiving historical branches as git tags.

### PR DESCRIPTION
**Pull request by carltongibson**
Suggested docs adjustment for moving old branches to tags.

### CODE REVIEW

**Comment by carltongibson** [comment]
Let me have a look...

**Comment by carltongibson** [comment]
OK, bigger rewrite...

File: docs/internals/git.txt [hunk 1]
```diff
@@ -146,13 +147,16 @@ become part of an official release, but others do not; in either case, there
 comes a time when the branch is no longer being actively worked on by any
 developer. At this point the branch is considered closed.
 
-Unfortunately, Django used to be maintained with the Subversion revision
+Django used to be maintained with the Subversion revision
 control system, that has no standard way of indicating this. As a workaround,
 branches of Django which are closed and no longer maintained were moved into
 ``attic``.
```
Comment (felixxm) [inline docs/internals/git.txt]: Wrap paragraph at 79 chars :thinking:

Comment (carltongibson) [reply docs/internals/git.txt]: I'll check.

**Comment by carltongibson** [comment]
> I wonder if "grafts" workaround will still work after removing branches 🤔 .

I'll give it a quick try. 

I suspect no-one, like no-one is still interested to see the merge commits for those branches. (Not in these `--rebase` days...)

**Comment by carltongibson** [comment]
Hi @ramiro. Can I ask your opinion here? 

Looking into `grafts`, it turns out they're now deprecated since git 2.18. There is an updated option with `git replace --convert-graft-file` but, looking into it, I wonder if [the paragraphs about this](https://github.com/django/django/blob/33d892f003ca814b99e00542d38f29478395cc97/docs/internals/git.txt#L199-L220) still have value? 

Is anyone worried about the extra merge commit history from 7-114 years ago? If anyone is, would we be better served dropping that discussion from the live page, leaving it accessible in the file commit history, for those folks? 

Can we remove these paragraphs at this point do you think?

Thanks!

File: docs/internals/git.txt (hunk 1 above)
Comment (carltongibson) [reply docs/internals/git.txt]: Good catch! Thank you!

**Comment by ramiro** [comment]
Hi Carlson!

> Looking into `grafts`, it turns out they're now deprecated since git 2.18. There is an updated option with `git replace --convert-graft-file`

I had in my TODO to re-try this conversion as I assumed we could start using a feature present in  GIt since 2014. My first attempt failed but I assume it was an error in my part. This would be nice to have so that information is part of the things which gets propagated when one clones the repository and there is no need to remind us/people to copy the `grafts` file manually just for the improbable case one wants to examine merge of these branchs for historical/archaelogical reasons.

> but, looking into it, I wonder if [the paragraphs about this](https://github.com/django/django/blob/33d892f003ca814b99e00542d38f29478395cc97/docs/internals/git.txt#L199-L220) still have value?

IMHO at this point, having that info has no value as it probably had at the time Adrian migrated us from SVN to Git back in 2012.
> Can we remove these paragraphs at this point do you think?

Agreed.

**Comment by carltongibson** [comment]
Hi @ramiro. Thanks for the quick reply. As such, I shall remove those paragraphs. 👍 

I'll then merge this, and migrate the branches to tags, tomorrow, as it's getting late and I'd rather do it with a clear head. 🙂

Have a great day/evening!

**Comment by pope1ni** [comment]
I just thought of something @carltongibson.

When you create all of these tags, can you backdate them so that they don't show up in GitHub as more recent than `3.0b1`, etc. That would be annoying! The following is from the `git-tag(1)` man page:

```
   On Backdating Tags
       If you have imported some changes from another VCS and would like to add tags for major releases of your work, it is useful
       to be able to specify the date to embed inside of the tag object; such data in the tag object affects, for example, the
       ordering of tags in the gitweb interface.

       To set the date used in future tag objects, set the environment variable GIT_COMMITTER_DATE (see the later discussion of
       possible values; the most common form is "YYYY-MM-DD HH:MM").

       For example:

           $ GIT_COMMITTER_DATE="2006-10-02 10:31" git tag -s v1.0.1
```

I suggest that we just use the commit date from the head of each branch for each new tag.

**Comment by carltongibson** [comment]
Hey @pope1ni. Already in hand, but good thank you! 👍 

If you'd like to sanity check, that would be great: https://gist.github.com/carltongibson/dec8597cf15d99e082c58413d83daad8

**Pull request merged**